### PR TITLE
Refactoring wind params

### PR DESCRIPTION
**Pull request by setmar**
Wind parameters are now collected in a python class and C struct (OpenCL), required minimal code changes, and no code changes in notebooks, to ensure backward compatibility. (This approach might be used for other parameters as well. If complex data types are to be included, some extra steps are needed, like packing of the C struct.)

Reading wind parameters from nc-file will (still) not work, and should be included in a later PR.

### CODE REVIEW

**Comment by setmar** [comment]
[Refactor the Wind class, so that it is clear which parameters each wind type must provide](https://trello.com/c/CI5Zmmwg/43-refactor-the-wind-class-so-that-it-is-clear-which-parameters-each-wind-type-must-provide)

File: python/SWESimulators/Simulator.py [hunk 1]
```diff
@@ -76,6 +76,10 @@ def __init__(self, \
         self.offset_x = offset_x
         self.offset_y = offset_y
         
+        # Upload wind stress params to device
```
Comment (havahol) [inline python/SWESimulators/Simulator.py]: Should this be a function in the WindStress class?

**Comment by havahol** [comment]
Examples of use of UniformAlongShoreWindStress can be found in Reproductions.ipynb, and fails in the constructor of UniformAlongShoreWindStress, due to missing parameter rho_air (it seems).
You will also need to change (h, B) to (eta, H) for KP07 and CDKLM16 to run all test cases in this notebook.

**Comment by setmar** [comment]
Bug fixed.

Common.WindStressParams() is implemented as a utility function, returning a WindStress object. I suggest we keep the signatures of the Simulator-classes as they are, to avoid changing nearly all notebooks at once. Since Common.WindStressParams() is marked as deprecated, we will remember to refactor as we "touch" the different notebooks.

**Comment by setmar** [comment]
Signatures of the Simulator-classes are changed to using BaseWindStress.NoWindStress() as default argument, since this did actually not imply changing all notebooks.

**Pull request merged**